packages/webpack.web.config.js

```diff
@@ -44,6 +36,22 @@ const webConfig = merge(baseConfig, {
             NODE_DEBUG: 'development',
             READABLE_STREAM: 'disable',
         }),
+        /** 
+         * HACK: the http resource fetcher breaks Web mode if imported, BUT we still dynamically import this module for non web mode
```

Comment:
why not use the exact name?

```suggestion
         * HACK: HttpResourceFetcher breaks Web mode if imported, BUT we still dynamically import this module for non web mode
```

Reply:
Good point, updated